.github/workflows/arm64.yml

```diff
@@ -15,19 +15,78 @@ env:
   DTAS_VERSION: v1.2.2
 
 jobs:
+  start-runner:
+    name: "Start self-hosted EC2 runner"
+    runs-on: ubuntu-latest
+    outputs:
+      label: ${{ steps.start-ec2-runner.outputs.label }}
+      ec2-instance-id: ${{ steps.start-ec2-runner.outputs.ec2-instance-id }}
+    steps:
+      - name: "Configure AWS credentials"
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_EC2_GITHUB_RUNNER_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_EC2_GITHUB_RUNNER_SECRET_ACCESS_KEY }}
+          aws-region: eu-west-2
+      - name: "Start EC2 runner"
+        id: start-ec2-runner
+        uses: machulav/ec2-github-runner@4e0303de215db88e1c489e07a15ca4d867f488ea
+        with:
+          mode: start
+          github-token: ${{ secrets.BOT_GITHUB_TOKEN }}
+          ec2-image-id: ami-0ec31c3e3b729bb6d
+          ec2-instance-type: c6g.2xlarge
+          subnet-id: subnet-0777c70a4cd9ce944
+          security-group-id: sg-0f89a325d9eb147cb
+          aws-resource-tags: >
+            [
+              {"Key": "Name", "Value": "arm64-acs-packaging-gh-runner"},
+              {"Key": "GitHubRepository", "Value": "${{ github.repository }}"}
```

Comment:
can you use some "standard" tags? https://alfresco.atlassian.net/wiki/spaces/TECH/pages/1266352366/AWS+tags+for+resource+identifications

Reply:
good idea, thanks for sharing the page as I wasn't aware of it. Will do 💯

Reply:
![tags](https://github.com/Alfresco/acs-packaging/assets/24280982/037bf09a-85f5-4616-a7b8-f1d0a941a082)